src/index.js

```diff
@@ -1,6 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import App from 'src/App';
+import store from 'src/store';
 
-import App from './components/App';
-
-ReactDOM.render(<App />, document.getElementById('root'))
\ No newline at end of file
+ReactDOM.render(
+  <Provider store={store}>
```

Comment:
as for me it should be part of App

generally index file is the point to the logic you have in another files

Reply:
Updated